### PR TITLE
doc: add minutes for meeting 30th september 2020

### PR DESCRIPTION
**Pull request by gireeshpunathil**
Fixes: https://github.com/nodejs/diagnostics/issues/433

### CODE REVIEW

**Review by RafaelGSS — APPROVED** [review]
LGTM.

**Review by Flarna — APPROVED** [review]

File: wg-meetings/2020-09-30.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=idW6Fe1fyXM
+* **GitHub Issue**: https://github.com/nodejs/diagnostics/issues/433
+
+## Present
+
+* Stephen Belanger (@qard)
+* Rafael Gonzaga (@rafaelgss) 
+* MIchael Dawson (@mhdawson)
```
Comment (mhdawson) [inline wg-meetings/2020-09-30.md:12]:
```suggestion
* Michael Dawson (@mhdawson)
```

File: wg-meetings/2020-09-30.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## Agenda
+
+## Announcements
+ 
+*Extracted from **diag-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* Proposal: mark AsyncResource and AsyncLocalStorage as stable [#35286](https://github.com/nodejs/node/issues/35286)
+  * AsyncResource any objects to that being considered stable
```
Comment (mhdawson) [inline wg-meetings/2020-09-30.md:28]:
```suggestion
* AsyncResource any objections to that being considered stable
```

**Review by mhdawson — APPROVED** [review]
LGTM with a few suggestions

File: wg-meetings/2020-09-30.md (hunk 1 above)
Comment (gireeshpunathil) [reply wg-meetings/2020-09-30.md]: fixed

File: wg-meetings/2020-09-30.md (hunk 2 above)
Comment (gireeshpunathil) [reply wg-meetings/2020-09-30.md]: fixed

**Comment by gireeshpunathil** [comment]
landed in 0ed4048143316f62d596eb715c340b9c23ae82f2

**Pull request merged**